File: JavaScript/tabla_verdad.js
function tablaVerdad() {
    console.log("Tabla de la verdad para la expresión (A AND B) OR (NOT A)");
    console.log("A\tB\tResultado");
    const valores = [false, true];
    for (let A of valores) {
        for (let B of valores) {
        const resultado = (A && B) || (!A);
        console.log(A + "\t" + B + "\t" + resultado);
        }
    }
}

tablaVerdad();

File: python/tabla_verdad.py
def tabla_verdad():
    print("Tabla de la verdad para la expresión (A AND B) OR (NOT A)")
    print("A\tB\tResultado")
    for A in [False, True]:
        for B in [False, True]:
            resultado = (A and B) or (not A)
            print(A, "\t", B, "\t", resultado)

tabla_verdad()
